components/Navbar/NavItem.js

```diff
@@ -8,7 +8,7 @@ const Pages = [
 ];
 function index() {
   return Pages.map((item, i) => (
-    <div key={item}>
+    <div key={item.title}>
```

Comment:
Can't we just use `i` here?

Reply:
I think we can but I get used to avoiding using `i` as key whenever I can. shall I change this here?